package.json

```diff
@@ -130,6 +130,7 @@
     "setupFiles": [
       "<rootDir>/jestMockSetup.js"
     ],
-    "cacheDirectory": ".jest/cache"
+    "cacheDirectory": ".jest/cache",
+    "testMatch": ["**/*.test.js"]
```

Comment:
I'm defining the test match because I don't want to run jest on auxiliary files for test.